src/analyser.js

```diff
@@ -0,0 +1,54 @@
+/*
+ * backpack-usage-analyser
+ *
+ * Copyright 2018-present Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const fs = require('fs');
+const path = require('path');
+const util = require('util');
+const globby = require('globby');
+
+const readFile = util.promisify(fs.readFile);
+
+const config = require('./config');
+
+const availablePlatforms = Object.keys(config);
+
+module.exports = async (platform) => {
+  if (availablePlatforms.indexOf(platform) === -1) {
```

Comment:
__Nitpick:__ a `Set` and using `has` is nicer for this kind of thing

Reply:
How about `availablePlatforms.includes` instead? If i use a `Set` then i'll have to refactor the following code:
```
console.error(`😱  Unrecognised platform, please specify one of: ${availablePlatforms.join(', ')}`);
```

Reply:
Yeah sounds good also noticed you are getting the keys from an object here so creating a set is not great anyway